framework/queries/postgres_db.py

```diff
@@ -99,3 +100,51 @@ def get_random_users(self, quantity: int = 1) -> List[dict]:
             """
         )
         return response
+
+    def create_user(self, user: dict) -> None:
+        """Inserting user into database
+
+        Args:
+            user: user data
+        """
+        self.db.execute(
+            f"""
+                INSERT INTO public.user_details(id
+                    , first_name
+                    , last_name
+                    , stripe_customer_token
+                    , email, password
+                    , address_id
+                    , account_non_expired
+                    , account_non_locked
+                    , credentials_non_expired
+                    , enabled
+                )
+                VALUES ('{user["id"]}'
+                    , '{user["first_name"]}'
+                    , '{user["last_name"]}'
+                    , null, '{user["email"]}'
+                    , '{user["hashed_password"]}'
+                    , null
+                    , true
+                    , true
+                    , true
+                    , true
+                );
+            """
+        )
+
+    def create_random_users(self, quantity: int = 1) -> List[dict]:
+        """Creating random user(s)
+
+        Args:
+            quantity: number of random users
+        """
+        users = [self.create_user(generate_user()) for _ in range(quantity)]
+
+        return users
+
+    def delete_user(self, user_id: str) -> None:
+        """Deletes a user from the database based on user ID"""
```

Comment:
can you add a description to the user_id parameter?

Reply:
Fixed.